### PR TITLE
Encrypted traffic metadata

### PR DESCRIPTION
**Pull request by spendletonliveaction**
This is a test for the feature https://redmine.openinfosecfoundation.org/issues/5372. The associated PR for that feature is here: https://github.com/OISF/suricata/pull/7415

### CODE REVIEW

File: tests/encrypted-traffic-metadata/test.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+checks:
+  - filter:
+      # The number of records this filter should match.
+      count: 1
+      # The fields to match on.
+      match:
+        # Example match on event_type:
+        event_type: flow
+        # Example match on array item:
+        flow.etm.entropy: 7.945903301239014
```
Comment (inashivb) [inline tests/encrypted-traffic-metadata/test.yaml:14]: Hi. Thank you for this test. :)
I see that you've added a bunch of fields to the schema (https://github.com/OISF/suricata/pull/7415/files#diff-fe105e2d0f7fb994ce462f504d65673950ccbba1ad8c106899aa560ca591a0d2R1277), is it possible to have an event with all/most of those fields?
Multiple tests are also welcome.
You can use our createst script: https://github.com/OISF/suricata-verify/blob/master/createst.py to auto generate them.
Instructions on its usage are in the README of this project.

**Comment by spendletonliveaction** [comment]
Thanks for looking at this. I added 3 tests total (one for HTTPS traffic, one for UDP traffic, and one for RDP traffic).

File: tests/encrypted-traffic-metadata/test.yaml (hunk 1 above)
Comment (spendletonliveaction) [reply tests/encrypted-traffic-metadata/test.yaml]: I added 3 tests. One for https, one for udp and one for rdp. I also changed the tests to test for all new fields.

**Review by inashivb — CHANGES_REQUESTED** [review]
Hello, could you please squash the commits that belong to the same test together? Thank you!

**Comment by jasonish** [comment]
Closing as the associated Suricata PR has been closed due to inactivity: https://github.com/OISF/suricata/pull/7415#issuecomment-1535950950

**Pull request closed without merging**